src/components/organisms/EntityDetails/EntityDetailsContainer/EntityDetailsContainer.tsx

```diff
@@ -177,28 +177,15 @@ const EntityDetailsContainer: React.FC<EntityDetailsBlueprint> = props => {
   };
 
   useEffect(() => {
-    if (execId && executionsList && executionsList?.results) {
```

Comment:
Does this feature work without this code?

Reply:
I cannot find any feature that would be broken after removal of this code. If you can verify this I would greatly appreciate.